packages/angular/projects/clr-angular/src/forms/common/if-control-state/if-success.ts

```diff
@@ -3,13 +3,15 @@
  * This software is released under MIT license.
  * The full license information can be found in LICENSE in the root directory of this project.
  */
-import { Directive, Optional, TemplateRef, ViewContainerRef } from '@angular/core';
+import { Directive, Optional, TemplateRef, ViewContainerRef, Input } from '@angular/core';
 import { NgControlService } from '../providers/ng-control.service';
 import { IfControlStateService, CONTROL_STATE } from './if-control-state.service';
 import { AbstractIfState } from './abstract-if-state';
 
 @Directive({ selector: '[clrIfSuccess]' })
 export class ClrIfSuccess extends AbstractIfState {
+  @Input('clrIfSuccess') success: string;
```

Comment:
```
Can't bind to 'clrIfSuccess' since it isn't a known property of 'clr-control-success'.
```
this is no longer needed cause this type of functionality is not implemented yet. Will leave it for the next iteration.